Error while launching DLC

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I tried launching the DLC GUI but I'm getting the following error:

wxAssertionError                          Traceback (most recent call last)
<ipython-input-2-1ced4355ffc5> in <module>
----> 1 deeplabcut.launch_dlc()

~\anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\gui\launch_script.py in launch_dlc()
     43 def launch_dlc():
     44     app = wx.App()
---> 45     frame = MainFrame().Show()
     46     app.MainLoop()

~\anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\gui\launch_script.py in __init__(self)
     25         media_path = os.path.join(dlcparent_path, "gui", "media")
     26         logo = os.path.join(media_path, "logo.png")
---> 27         self.SetIcon(wx.Icon(logo))
     28         # Here we create a panel and a notebook on the panel
     29         self.panel = wx.Panel(self)

wxAssertionError: C++ assertion "strcmp(setlocale(LC_ALL, NULL), "C") == 0" failed at ..\..\src\common\intl.cpp(1579) in wxLocale::GetInfo(): You probably called setlocale() directly instead of using wxLocale and now there is a mismatch between C/C++ and Windows locale.
Things are going to break, please only change locale by creating wxLocale objects to avoid this!

### Operating System

operating system Windows 10

### DeepLabCut version

dlc version 2.2rc3

### DeepLabCut mode

single animal

### Device type

gpu RTX 3060

### Steps To Reproduce

1. In the environment DEEPLABCUT
2. Type ipython --> import deeplabcut --> deeplabcut.launch_dlc()

### Relevant log output

```shell
wxAssertionError                          Traceback (most recent call last)
<ipython-input-2-1ced4355ffc5> in <module>
----> 1 deeplabcut.launch_dlc()

~\anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\gui\launch_script.py in launch_dlc()
     43 def launch_dlc():
     44     app = wx.App()
---> 45     frame = MainFrame().Show()
     46     app.MainLoop()

~\anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\gui\launch_script.py in __init__(self)
     25         media_path = os.path.join(dlcparent_path, "gui", "media")
     26         logo = os.path.join(media_path, "logo.png")
---> 27         self.SetIcon(wx.Icon(logo))
     28         # Here we create a panel and a notebook on the panel
     29         self.panel = wx.Panel(self)

wxAssertionError: C++ assertion "strcmp(setlocale(LC_ALL, NULL), "C") == 0" failed at ..\..\src\common\intl.cpp(1579) in wxLocale::GetInfo(): You probably called setlocale() directly instead of using wxLocale and now there is a mismatch between C/C++ and Windows locale.
Things are going to break, please only change locale by creating wxLocale objects to avoid this!
```


### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

--- a/deeplabcut/gui/launch_script.py
+++ b/deeplabcut/gui/launch_script.py
@@ -42,5 +42,6 @@ class MainFrame(BaseFrame):
 
 def launch_dlc():
     app = wx.App()
+    app.locale = wx.Locale(wx.LANGUAGE_ENGLISH)
     frame = MainFrame().Show()
     app.MainLoop()

